include/sway/config.h

```diff
@@ -499,6 +499,8 @@ struct sway_config {
 	int gaps_inner;
 	struct side_gaps gaps_outer;
 
+    bool user_config_path;
```

Comment:
This can get moved down into the group below. Probably in between `config_chain` and `current_config_path`.